src/OpenStudy/study_openinterface.jl

```diff
@@ -41,6 +41,60 @@ end
 # TODO: rebuild "raw" stabilizing data types
 # TODO fuel consumption updater
 
+mutable struct DataIndex
+    # `index` takes a `reference_id` as key and returns a pair
+    # containing the collection from which the referenced item
+    # belongs but also its index in the vector of instances of
+    # the collection.
+    index::Dict{Int,Tuple{String,Int}}
+
+    # This should be equal to `Set{Int}(keys(data_index.index))`
+    key_set::Set{Int}    
```

Comment:
Why do you need it to be a set?

Reply:
This became useless now, we must drop this field.